.vscode/deno_resolve_npm_imports.json

```diff
@@ -8,6 +8,9 @@
   ],
   "imports": {
     "@remix-run/server-runtime": "https://esm.sh/@remix-run/server-runtime@nightly",
+    "@remix-run/deno": "../packages/remix-deno/index.ts",
+    "@remix-run/deno/": "../packages/remix-deno/",
+    "@netlify/edge-functions": "https://edge.netlify.com",
```

Comment:
I think we should add these to the template's `.vscode/resolve_npm_imports.json` as well? 🤔

Reply:
No, they're compiled away by then

Reply:
Since `@remix-run/netlify-edge` isn't compiled, these ones aren't compiled as well, so we should include them in the template's `.vscode/resolve_npm_imports.json`, just like we do for the Deno template I guess?

CC/ @pcattori

Reply:
`@remix-run/netlify-edge` is compiled

Reply:
Since Deno supports TS, I think we don't need to compile `@remix-run/netlify-edge` either, just like we don't compile `@remix-run/deno` 🤔

@pcattori can probably give more insights in what's best here.

Reply:
It's imported by code that's then compiled by node and rollup. Using an import map won't make any difference, because rollup will ignore it and just load and bundle `@remix-run/deno`, `@netlify/edge-functions` and `@remix-run/server-runtime` from `node_modules` anyway. The Remix compiler always compiles the site's `server.ts`, so even if it's using imports that are Deno-friendly, that's misleading because it's all compiled into a bundled `server.js` file before Deno ever runs it. The exception is URL imports, which rollup ignores and passes-through unchanged in the generated code.

Reply:
I think we don't compile `@remix-run/deno` for better IDE support, but @pcattori can probably give more insights into the reasoning behind it.

Whatever we decide to do, I think we should streamline it in both the `deno` & `netlify-edge` template though.